conf/application-json-logger.xml

```diff
@@ -9,6 +9,8 @@
 
     <logger name="application.services.DataQueryServiceImpl" level="${logger.services.DataQueryService:-INFO}"/>
 
+    <logger name="application.services.DataUpdateService" level="${logger.services.DataUpdateService:-INFO}"/>
```

Comment:
I think the name should actually be application.services.DataUpdateServiceImpl

Reply:
Done